config/config.sample.php

```diff
@@ -2534,6 +2534,15 @@
  */
 'unified_search.enabled' => false,
 
+/**
+ * List of search providers that are allowed to be used in the unified search
+ *
+ * Defaults to ``[]`` (all providers allowed)
+ *
+ * For example: Set to ['files', 'settings'] to only allow file and settings search.
+ */
+'unified_search_providers_allowed' => [],
```

Comment:
We should keep the same format for all related options:

```suggestion
'unified_search.providers_allowed' => [],
```